.gitignore

```diff
@@ -47,6 +47,10 @@ lib/public/*
 !lib/public/.gitkeep
 !lib/public/docker-compose.public.yaml
 
+# gohan_elasticsearch
+lib/gohan/es_jvm_options/*
+!lib/gohan/es_jvm_options/.gitkeep
```

Comment:
what happens when the user runs gohan without anything in here; does it work, or should we have a default editable file here?

Reply:
It works with nothing in it as well. This dir binds to `/usr/share/elasticsearch/config/jvm.options.d/`, which is always empty by default . Only files with the `.options` suffix are read by ES there, so whatever is in .gitkeep is ignored.

https://www.elastic.co/guide/en/elasticsearch/reference/current/advanced-configuration.html#set-jvm-options